lib/interop.js

```diff
@@ -363,9 +363,18 @@ Interop.prototype.toUnifiedPlan = function(desc) {
     // In the iteration that follows, we use the cached Unified Plan (if it
     // exists) to assign mids to ssrcs.
 
+    var type;
+    if (desc.type === 'answer') {
+        type = 'offer';
+    } else if (desc.type === 'offer') {
+        type = 'answer';
+    } else {
+        throw new Error("Type '" + desc.type + "' not supported.");
+    }
+
```

Comment:
Hi @mparis, thanks again for all your work, I really appreciate it! I'm trying to understand the problem you're trying to fix here. In any case this is a little too risky to accept, at least as is.